spec/unit/remote_system_spec.rb

```diff
@@ -18,7 +18,15 @@
 require_relative "spec_helper"
 
 describe RemoteSystem do
-  let(:remote_system) { RemoteSystem.new("remotehost") }
+  let(:rsys) { RemoteSystem.new("remotehost") }
```

Comment:
Wouldn't all the overriden option hashes have to be called `options` to make that work?